mappings/net/minecraft/entity/ai/brain/Brain.mapping

```diff
@@ -37,7 +46,34 @@ CLASS net/minecraft/class_4095 net/minecraft/entity/ai/brain/Brain
 	METHOD method_18906 hasActivity (Lnet/minecraft/class_4168;)Z
 	METHOD method_18911 copy ()Lnet/minecraft/class_4095;
 	METHOD method_19542 tick (Lnet/minecraft/class_3218;Lnet/minecraft/class_1309;)V
+		ARG 1 world
+		ARG 2 entity
 	METHOD method_19544 updateSensors (Lnet/minecraft/class_3218;Lnet/minecraft/class_1309;)V
 	METHOD method_19545 updateTasks (Lnet/minecraft/class_3218;Lnet/minecraft/class_1309;)V
 	METHOD method_19948 isEmptyCollection (Ljava/lang/Object;)Z
 		ARG 1 value
+	METHOD method_24524 indexTaskList (ILcom/google/common/collect/ImmutableList;)Lcom/google/common/collect/ImmutableList;
+		ARG 1 n
```

Comment:
What's `n`?

Reply:
Like the starting index of the new task list indexes, I recall. Let me see again.